File: main_module/src/main/java/com/phone/main_module/main/view/IMainView.java
package com.phone.main_module.main.view;


import com.phone.common_library.base.IBaseView;

/**
 * author    : Urasaki
 * e-mail    : 1164688204@qq.com
 * date      : 2020/3/7 15:44
 * introduce :
 */


public interface IMainView extends IBaseView {

    void mainDataSuccess(String success);

    void mainDataError(String error);
}

File: module_main/src/main/java/com/phone/module_main/main/view/IMainView.kt
package com.phone.module_main.main.view

import com.phone.common_library.base.IBaseView

/**
 * author    : Urasaki
 * e-mail    : 1164688204@qq.com
 * date      : 2020/3/7 15:20
 * introduce :
 */

interface IMainView: IBaseView {

    fun mainDataSuccess(success: String)

    fun mainDataError(error: String)
}